.github/workflows/keepalived-release.yaml

```diff
@@ -0,0 +1,138 @@
+# This code is borrowed from https://github.com/kubernetes-sigs/cluster-api/blob/main/.github/workflows/release.yaml
```

Comment:
I doubt we will use this automation for the keepalived with release branches etc.